### PR TITLE
feat: Show incoming bridge placeholder in destination network

### PR DESCRIPTION
**Pull request by fpezzola**
# Name of the feature/issue
Bridging

## Description
This pr meant to show the incoming bridging transactions before we already know the destination transaction hash so that the user can see the "Incoming bridge" transaction in their activity list while it is pending.

Click the correct/s option/s

- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  This change requires a documentation update

https://user-images.githubusercontent.com/22504074/192754076-6d2fe986-b1e5-43e9-9927-e7c5bf6a1e4b.mov

### CODE REVIEW

File: packages/ui/src/components/transactions/TransactionItem.tsx [hunk 1]
```diff
@@ -595,6 +597,8 @@ const TransactionItem: React.FC<{
 
                         {status === TransactionStatus.SUBMITTED &&
                             metaType === MetaType.REGULAR &&
+                            transactionCategory !==
```
Comment (julianariel) [inline packages/ui/src/components/transactions/TransactionItem.tsx:600]: this conditional is getting a bit complex, shouldn't we move it to a variable that "explains" it?

**Review by julianariel — APPROVED** [review]
awesome work, this solves the problem in a clean way and improves the bridging experience a lot! 👑

File: packages/ui/src/components/transactions/TransactionItem.tsx (hunk 1 above)
Comment (fpezzola) [reply packages/ui/src/components/transactions/TransactionItem.tsx]: Done and removed some tornado logic

File: packages/background/src/controllers/ActivityListController.ts [hunk 2]
```diff
@@ -89,7 +91,45 @@ export class ActivityListController extends BaseController<IActivityListState> {
             selectedAddress
         );
 
-        const incomingBridgeTransactions =
+        //Incoming bridges that may or may not be converted into an incoming transaction for this network and account.
+        //If the bridge is completed successfully, then it will appear as a trasnaction gathered by the BridgingController.
+        //if the bridge failed, it will be filtered by the  tx.bridgeParams.status === BridgeStatus.PENDING condition
+        const pendingIncomingBridgePlaceholders =
```
[the remark this answers is not in the document]
Comment (fpezzola) [reply packages/background/src/controllers/ActivityListController.ts:95]: Done

**Pull request merged**